### PR TITLE
BugFix: adding app insights and logging to configuration settings

### PR DESCRIPTION
**Pull request by moiradillon12**
## Description
Adding azure monitor details (app insights & log analytics)  to configuration settings doc

## Related issues
[AB#75897](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/75897)
[AB#76021](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/76021)

## Testing
VS Preview

### CODE REVIEW

File: docs/how-to-guides/configure-dicom-server-settings.md [hunk 1]
```diff
@@ -49,16 +49,42 @@ If you would like to update your SQL Database tier:
 ![Configure SQL2](../images/configure-sql-2.png)
 1. Click **Apply**.
 
-## Additional Configuration Settings
+## Azure Monitor
 
-**Application Insights**: By default, Application Insights is deployed, but not enabled, when you deploy the Medical Imaging Server for DICOM to Azure. To update this setting, navigate to your App Service:
+[Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/overview) offers a variety of solutions to collect, analyze and act on telemetry, including Application Insights Log Analytics.
```
Comment (pengchen0692) [inline docs/how-to-guides/configure-dicom-server-settings.md:54]: Last time as I talked with Brandon , I learned that although appinsights is not enabled, our application is still using AppInsights for logging -- which is a little bit surprise to me. 
image


I'm not sure how we want to phrase this to customer, also not sure how much appinsights functionalities we are utilizing, could you check with Brandon who knows this much better to figure this out?

File: docs/how-to-guides/configure-dicom-server-settings.md [hunk 2]
```diff
@@ -49,16 +49,42 @@ If you would like to update your SQL Database tier:
...
 
-## Additional Configuration Settings
+## Azure Monitor
 
-**Application Insights**: By default, Application Insights is deployed, but not enabled, when you deploy the Medical Imaging Server for DICOM to Azure. To update this setting, navigate to your App Service:
+[Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/overview) offers a variety of solutions to collect, analyze and act on telemetry, including Application Insights Log Analytics.
 
+### Application Insights
+
+By default, Application Insights is deployed, but not enabled, when you deploy the Medical Imaging Server for DICOM to Azure. To update this setting:
```
Comment (pengchen0692) [inline docs/how-to-guides/configure-dicom-server-settings.md:58]: Last time as I talked with Brandon , I learned that although appinsights is not enabled, our application is still using AppInsights for logging -- which is a little bit surprise to me. 
image


I'm not sure how we want to phrase this to customer, also not sure how much appinsights functionalities we are utilizing, could you check with Brandon who knows this much better to figure this out?

Comment (brandonpollett) [reply docs/how-to-guides/configure-dicom-server-settings.md:58]: When you use the template to deploy, we default to deploy application insights. We wire this up using the following commands which wire up server side telemetry and our logger: 
https://github.com/microsoft/dicom-server/blob/b4240ae765d350f359ed34032a50cf73fbb70188/src/Microsoft.Health.Dicom.Web/Startup.cs#L57-L58

More information about this approach can be found about this approach [here](https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core) and [here](https://docs.microsoft.com/en-us/azure/azure-monitor/app/ilogger)

Comment (moiradillon12) [reply docs/how-to-guides/configure-dicom-server-settings.md]: @brandonpollett and @pengchen0692 I updated the text, let me know what you think!

File: docs/how-to-guides/configure-dicom-server-settings.md [hunk 3]
```diff
@@ -49,16 +49,44 @@ If you would like to update your SQL Database tier:
...
 
-## Additional Configuration Settings
+## Azure Monitor
 
-**Application Insights**: By default, Application Insights is deployed, but not enabled, when you deploy the Medical Imaging Server for DICOM to Azure. To update this setting, navigate to your App Service:
+[Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/overview) offers a variety of solutions to collect, analyze and act on telemetry, including Application Insights Log Analytics.
 
+### Application Insights
+
+Application Insights is deployed when you deploy an instance of the Medical Imaging Server for DIOCM to Azure. To enable and customize Application Insights:
```
Comment (brandonpollett) [inline docs/how-to-guides/configure-dicom-server-settings.md:58]: If you just take the defaults from our template, the variable `deployApplicationInsights` is set as `true `which will turn on app insights for you. You might want to take a look and see if following these steps provides anything additional/different, and if not call out that this is already done by default.

Comment (moiradillon12) [reply docs/how-to-guides/configure-dicom-server-settings.md:58]: If I use the default, deployApplicationInsights is set as true. However, once I navigate to the application insights page on my app service, app insights is not turned on. I am given the option to 'enable application insights without redeploying your code' by 'turning on app insights'.

Comment (moiradillon12) [reply docs/how-to-guides/configure-dicom-server-settings.md:58]: @brandonpollett just wanted to follow up here!

**Review by brandonpollett — APPROVED** [review]

**Pull request merged**